book/src/feature/worker_pool.md

```diff
@@ -0,0 +1,21 @@
+# Worker pool
```

Comment:
TODO(fzyzcjy): Refine doc after PR merges

(you can ignore this comment, just for reminder of myself)